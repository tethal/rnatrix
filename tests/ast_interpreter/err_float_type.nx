fun main() {
    print(float(null));
}
