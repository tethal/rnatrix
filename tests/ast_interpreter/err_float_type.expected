<string>:2:11: error: float cannot be applied to Null
    print(float(null));
          ^^^^^^^^^^^
